org.eclipse.scanning.api/src/org/eclipse/scanning/api/device/IScannableDeviceService.java

```diff
@@ -48,6 +50,8 @@
  */
 public interface IScannableDeviceService {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1214)